biomage/stage/stage.py

```diff
@@ -543,19 +564,26 @@ def stage(token, org, deployments):
     all_experiments = get_all_experiments(config["production-experiments-table"])
 
     manifest, sandbox_id = create_manifest(
-        templates, token, repo_to_ref=repo_to_ref, all_experiments=all_experiments
+        templates,
+        token,
+        repo_to_ref=repo_to_ref,
+        all_experiments=all_experiments,
+        auto=auto,
     )
 
-    print("select staging experiments")
-    experiment_ids_to_stage, staged_experiment_ids = select_staging_experiments(
-        sandbox_id, all_experiments, config
-    )
+    click.echo()
+    click.echo(f"Sandbox ID: {sandbox_id}")
+
+    experiment_ids_to_stage, staged_experiment_ids = [], []
+    # do not stage automatically any experiment on auto mode
+    if not auto:
+        click.echo("select staging experiments")
+        experiment_ids_to_stage, staged_experiment_ids = select_staging_experiments(
+            sandbox_id, all_experiments, config
+        )
 
     if len(experiment_ids_to_stage) == 0:
-        click.echo(
-            "No experiments chosen. "
-            "Skipping creation of isolated staging environment."
-        )
+        click.echo("No experiments chosen. ")
```

Comment:
Why remove the second line? Was it outdated? Also trailing space.

Reply:
because it's misleading, the isolated staging environment exists regardless of whether experiments are copied to it or not. 

removed the trailing space.